scripts/docker_test_ml.sh

```diff
@@ -6,6 +6,7 @@ source scripts/l4t_version.sh
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 TEST_MOUNT="$ROOT/../test:/test"
 CONTAINERS=${1:-"all"}
+DOCKER_REPO=$2
```

Comment:
@atharvanan1 should we add a note to https://github.com/atharvanan1/jetson-containers#testing-the-containers regarding testing prebuilt containers?

Reply:
That's a good one. I'll implement that.

Reply:
Done